<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLogin</class>
 <widget class="QWidget" name="FormLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_right">
   <property name="geometry">
    <rect>
     <x>488</x>
     <y>60</y>
     <width>351</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-top-right-radius:10px;
border-bottom-right-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackLR">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>301</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="edit_user_logPage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);</string>
      </property>
      <property name="placeholderText">
       <string>账号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_pwd_logPage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_log">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_log{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border:2px solid rgb(0,0,0);
	border-radius:8px;
}
#btn_log:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
#btn_log:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
      </property>
      <property name="text">
       <string>登  录</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>10</y>
       <width>421</width>
       <height>301</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="edit_user_regPage">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);</string>
      </property>
      <property name="placeholderText">
       <string>账号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_pwd_regPage">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(2, 198, 211);
border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_reg{
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
	border:2px solid rgb(0,0,0);
	border-radius:8px;
}
#btn_reg:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
#btn_reg:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
      </property>
      <property name="text">
       <string>注 册</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_cfg_regPage">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>291</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0,0,0);
border-radius:8px;
border-color: rgb(2, 198, 211);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>确认密码：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_left">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>431</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:10px;
background-color: rgb(85, 255, 0);
border-bottom-left-radius:10px;
border-image: url(:/res/face.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widgetLR" native="true">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_log_sw">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{
	border:none;
}
#pushButton:focus{
	color:rgb(180,180,180);
}</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_reg_sw">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_2{
	border:none;
}
#pushButton_2:focus{
	color:rgb(180,180,180);
}</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>813</x>
     <y>64</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/del.png</normaloff>:/res/del.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
